src/data/dataset_arms.py

```diff
@@ -125,7 +141,23 @@ def unique_output(
                 18,
                 19,
             ]
-            # Keep [0,2] legs, 4 = one values fields (pelvis + chest), 8 = third iso, [16,17,18,19] head, [20,22] = arms
+
+            if (
+                self.output == 19
```

Comment:
Can we add a global config in config.json?

Reply:
I think it would be pointless a global config for the outoput dimension, since the network heavily relies on specific and encoding and decoding of input and output. Therefor if you were to change the dimensionality without adjustment to the script, it would simply not work. This being said i can still make it less dependent on the magic number by using already existing global config.